tutorial/sample-problems/Misc/ChemicalReaction.pg

```diff
@@ -11,28 +11,25 @@
 ## MO(1)
 ## KEYWORDS('chemical reaction', 'template')
 
-# References:
-# http://webwork.maa.org/pod/pg_TRUNK/macros/contextReaction.pl.html
-# http://webwork.maa.org/moodle/mod/forum/discuss.php?d=449
-
 #:% name = Chemical Reaction
 #:% type = Sample
 #:% subject = chemistry
 #:% categories = [chemistry]
 
 #:% section = preamble
-#: Load `contextReaction.pl` to put chemical reactions/equations in a `Compute`
+#: Load PODLINK('contextReaction.pl)` to be able to use chemical reaction
```

Comment:
This needs to be `Load PODLINK('contextReaction.pl') to be ...`

Reply:
Thanks.  I fixed it.